tests/your_first_contract.rs

```diff
@@ -0,0 +1,134 @@
+#![deny(warnings)]
+
+use cairo_rs::vm::runners::cairo_runner::ExecutionResources;
+use felt::Felt;
+use num_traits::Zero;
+use starknet_rs::{
+    business_logic::{
+        execution::{
+            execution_entry_point::ExecutionEntryPoint,
+            objects::{CallInfo, CallType, TransactionExecutionContext},
+        },
+        fact_state::{
+            contract_state::ContractState, in_memory_state_reader::InMemoryStateReader,
+            state::ExecutionResourcesManager,
+        },
+        state::cached_state::CachedState,
+    },
+    definitions::{constants::TRANSACTION_VERSION, general_config::StarknetGeneralConfig},
+    services::api::contract_class::{ContractClass, EntryPointType},
+    starknet_storage::dict_storage::DictStorage,
+    utils::{calculate_sn_keccak, Address},
+};
+use std::{
+    collections::{HashMap, HashSet},
+    path::PathBuf,
+};
+
+#[test]
+fn your_first_contract_integration_test() {
+    // ---------------------------------------------------------
+    //  Create program and entry point types for contract class
+    // ---------------------------------------------------------
+
+    let path = PathBuf::from("tests/your_first_contract.json");
+    let contract_class = ContractClass::try_from(path).unwrap();
+    let entry_points_by_type = contract_class.entry_points_by_type().clone();
+
+    let increase_balance_selector = entry_points_by_type
```

Comment:
add a comment with the name of the entrypoint you are going to execute

Reply:
Resolved https://github.com/lambdaclass/starknet_in_rust/pull/157/commits/4c62931831bb3355f68a5efb6f91a2eafe46eb50